File: python/PRAK602-2310817310004-MSAMILRENDYNORSALEH.py
def hitungzetzu(ruangan, jumlahzetsu):
    for i in range(ruangan):
        jumlahzetsu[i] *= (i + 1)
        print(jumlahzetsu[i], end=' ')

if __name__ == "__main__":
    ruangan = int(input())
    
    jumlahzetsu = list(map(int, input().split()))

    hitungzetzu(ruangan, jumlahzetsu)
    print()

File: c/PRAK602-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>

void hitungzetzu(int ruangan, int jumlahzetzu[]) {
    for (int i = 0; i < ruangan; i++) {
        jumlahzetzu[i] *= (i + 1);
        printf("%d ", jumlahzetzu[i]);
    }
}

int main() {
    int ruangan;
    printf("");
    scanf("%d", &ruangan);

    int jumlahzetzu[ruangan];
    printf("");
    for (int i = 0; i < ruangan; i++) {
        scanf("%d", &jumlahzetzu[i]);
    }
    hitungzetzu(ruangan, jumlahzetzu);
    printf("\n");

    return 0;
}
